### PR TITLE
[backport] Always run cythonize on sdist installation

### PR DESCRIPTION
**Pull request by emcastillo**
Backport of #4619

### CODE REVIEW

**Comment by emcastillo** [comment]
[automatic post] Jenkins, test this please.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel", "Cython>=0.28.0", "fastrlock>=0.5"]
```
Comment (kmaehashi) [inline pyproject.toml:2]:
```suggestion
requires = ["setuptools", "wheel", "Cython>=0.28.0", "fastrlock>=0.3"]
```

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Pull request merged**